portfolio.css

```diff
@@ -763,3 +749,351 @@ textarea {
   position: sticky;
   right: 35px;
 }
+
+@media (min-width: 600px) {
```

Comment:
- [ ] Please fix the media queries for the Desktop breakpoint, the value for min-width property should be in range of 992px - 1100px.